otx/cli/utils/multi_gpu.py

```diff
@@ -152,15 +153,22 @@ def is_available(self) -> bool:
 
     def setup_multi_gpu_train(
         self,
-        output_path: str,
+        output_path: Optional[str] = None,
         optimized_hyper_parameters: Optional[ConfigurableParameters] = None,
-    ):
+    ) -> str:
         """Carry out what should be done to run multi GPU training.
 
         Args:
             output_path (str): output path where task output are saved.
             optimized_hyper_parameters (ConfigurableParameters or None): hyper parameters reflecting HPO result.
+
+        Returns:
+            str:
+                If output_path is None, make a temporary directory and return it.
         """
+        if output_path is None:
+            output_path = tempfile.mkdtemp(prefix="OTX-multi-gpu-")
```

Comment:
Is it will be not removed automatically via ContextManager?

Reply:
No. I will update it to be removed automatically. Thanks for pointing it!

Reply:
I fixed.